modules/RsIskraMeter/src/main.rs

```diff
@@ -813,6 +819,148 @@ impl ReadyState {
             }
         }
     }
+
+    /// Write custom string to LCD display registers
+    ///
+    /// Generic function to write strings to either the main LCD register or label register.
+    /// Non-printable values are replaced with empty space by the meter.
+    /// Also sets bit 3 in the LCD parameters register (47062) to enable custom string display.
+    ///
+    /// # Arguments
+    /// * `custom_string` - The string to display on the LCD
+    /// * `register_address` - The starting register address (47063 for main, 47067 for label)
+    /// * `max_chars` - Maximum number of characters (8 for main, 4 for label)
+    /// * `num_registers` - Number of registers to write (4 for main, 2 for label)
+    fn write_lcd_register(
+        &self,
+        custom_string: &str,
```

Comment:
where do we check if the string contains only valid chars? Or does then the write command fail?